gcloud-java-storage/src/main/java/com/google/gcloud/storage/Blob.java

```diff
@@ -267,18 +285,18 @@ public Storage storage() {
    * {@code infos.length > 1} a batch request is used to fetch blobs.
    *
    * @param storage the storage service used to issue the request
-   * @param infos the blobs to get
+   * @param blobs the blobs to get
    * @return an immutable list of {@code Blob} objects. If a blob does not exist or access to it has
-   * been denied the corresponding item in the list is {@code null}.
+   *     been denied the corresponding item in the list is {@code null}.
    * @throws StorageException upon failure
    */
-  public static List<Blob> get(final Storage storage, BlobInfo... infos) {
+  public static List<Blob> get(final Storage storage, BlobId... blobs) {
     checkNotNull(storage);
-    checkNotNull(infos);
-    if (infos.length == 0) {
+    checkNotNull(blobs);
```

Comment:
should we also change other places in this files that used to pass `info.bucket(), info.name()` and pass `info.blobId()` instead?